### PR TITLE
Span: Added castTo

### PR DESCRIPTION
**Pull request by JanosGit**

### CODE REVIEW

File: include/vctr/Miscellaneous/StdRatioHelpers.h [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    using Type = std::ratio<Ratio::num / gcd, Ratio::den / gcd>;
+};
+
+/** Reduces a std::ratio to its lowest form */
+template <is::stdRatio Ratio>
+using ReducedToLowestForm = typename detail::ReducedToLowestFormHelper<Ratio>::Type;
+
+/** Multiplies a std::ratio with an integer number.
+
+    The return type is the type returned by std::div called for the singed
```
Comment (iromur) [inline include/vctr/Miscellaneous/StdRatioHelpers.h]: Sing, sing, sing?

File: include/vctr/Containers/Span.h [hunk 2]
```diff
@@ -181,6 +181,29 @@ class Span : public VctrBase<ElementType, std::span<ElementType, extent>, extent
         Vctr::assignExpressionTemplate (expression);
     }
 
+    /** Casts this Span to a span of a different type by applyling a reinterpret cast to the data.
+
+        The size ratio between the source and the destination type multiplied by the source type
```
Comment (iromur) [inline include/vctr/Containers/Span.h]: "source size"?

**Review by iromur — APPROVED** [review]
Looks good (after some head explosions)!
Minor comment comments.

File: include/vctr/Containers/Span.h (hunk 2 above)
Comment (JanosGit) [reply include/vctr/Containers/Span.h]: of course...

**Pull request merged**